packages/repository-json-schema/.gitignore

```diff
@@ -0,0 +1,3 @@
+*.tgz
+dist*
+package
```

Comment:
I believe we are using a shared top-level `.gitignore` file, please remove this package-specific one.